### PR TITLE
Feature/107 fix bugs security issues and maintenance issues in mccl and mcpp

### PR DESCRIPTION
**Pull request by lmalenfant**

### CODE REVIEW

File: src/Vts.MonteCarlo.CommandLineApplication/Program.cs
Comment (hayakawa16) [inline src/Vts.MonteCarlo.CommandLineApplication/Program.cs:1]: Why is suggested update to local variable CPUCount name cpuCount and not cPuCount?  I found an example of local variable in pMCDatabase.cs of pMCDatabase and suggested name is pMcDatabase.

File: src/Vts.MonteCarlo.PostProcessor/Program.cs
Comment (hayakawa16) [inline src/Vts.MonteCarlo.PostProcessor/Program.cs:1]: Thanks!

File: src/Vts.MonteCarlo.CommandLineApplication/Program.cs
Comment (lmalenfant) [reply src/Vts.MonteCarlo.CommandLineApplication/Program.cs:1]: CPU is an acronym like API and are considered words like Count and Database, putting an uppercase P in the middle is not right, I guess it made that suggestion incorrectly, it should be pmcDatabase in camel case and PmcDatabase in Pascal case.

Comment (hayakawa16) [reply src/Vts.MonteCarlo.CommandLineApplication/Program.cs:1]: I see.  Thanks for your response.

**Review by hayakawa16 — APPROVED** [review]
Looks good.

**Pull request merged**